string2int/string2int/string2int.cpp

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+
+int* str2int(const char* string)
+{
+	//decleration of index representing the position in the string
+	int i = 0;
+	int Negative = 0;
+
+	if (string[i] == 45)
+	{
+		Negative = 1;
+	}
+
+	//finding the amount of characters in the string
+	while (string[i] != '\0')
```

Comment:
A safer way to do this is to receive the length of the string as a function parameter